Build failed:

```
builder for '/nix/store/8pm4jr9cji1yfk6gfbi1bazkssbgbh1a-mariadb-server-10.3.18.drv' failed with exit code 2; last 10 log lines:
        _auth_sha256_client in sha256_pw.c.o
    "_RSA_public_encrypt", referenced from:
        _auth_sha256_client in sha256_pw.c.o
    "_RSA_size", referenced from:
        _auth_sha256_client in sha256_pw.c.o
  ld: symbol(s) not found for architecture x86_64
  clang-7: error: linker command failed with exit code 1 (use -v to see invocation)
  make[2]: *** [libmariadb/CMakeFiles/sha256_password.dir/build.make:84: libmariadb/sha256_password.so] Error 1
  make[1]: *** [CMakeFiles/Makefile2:605: libmariadb/CMakeFiles/sha256_password.dir/all] Error 2
  make: *** [Makefile:152: all] Error 2
cannot build derivation '/nix/store/58axybx0k1gg4svqqh4k25wyaavirj8s-env.drv': 1 dependencies couldn't be built
[0 built (1 failed), 1 copied (68.6 MiB), 62.7 MiB DL]
error: build of '/nix/store/58axybx0k1gg4svqqh4k25wyaavirj8s-env.drv' failed
1 package failed to build:
mariadb
```

--- a/pkgs/servers/sql/mariadb/default.nix
+++ b/pkgs/servers/sql/mariadb/default.nix
@@ -2,8 +2,7 @@
 , libiconv, openssl, pcre, boost, judy, bison, libxml2, libkrb5, linux-pam, curl
 , libaio, libevent, jemalloc, cracklib, systemd, numactl, perl
 , fixDarwinDylibNames, cctools, CoreServices
-, asio, buildEnv, check, scons
-, less, fetchpatch
+, asio, buildEnv, check, scons, less
 , withoutClient ? false
 }:
 
@@ -21,14 +20,14 @@ mariadb = server // {
 };
 
 common = rec { # attributes common to both builds
-  version = "10.3.18";
+  version = "10.3.20";
 
   src = fetchurl {
     urls = [
       "https://downloads.mariadb.org/f/mariadb-${version}/source/mariadb-${version}.tar.gz"
       "https://downloads.mariadb.com/MariaDB/mariadb-${version}/source/mariadb-${version}.tar.gz"
     ];
-    sha256 = "1p6yvmahnkmsz50zjzp20ak7jzbqysly5bdl51nnrngrbfl6qib9";
+    sha256 = "14n4zfpwhvafz02r95bidmkwq2bz9jj3safqni1h21jfd0nqz0ak";
     name   = "mariadb-${version}.tar.gz";
   };
 
@@ -46,12 +45,7 @@ common = rec { # attributes common to both builds
   patches = [
     ./cmake-includedir.patch
     ./cmake-libmariadb-includedir.patch
-  ] ++ optional stdenv.hostPlatform.isDarwin (fetchpatch {
-    url = "https://github.com/MariaDB/mariadb-connector-c/commit/ee91b2c98a63acb787114dee4f2694e154630928.patch";
-    extraPrefix = "libmariadb/";
-    sha256 = "06i865zwyhs9fvrgmargzn09pbg1cmably3c4wifd241bj8ig8qk";
-    stripLen = 1;
-  });
+  ];
 
   cmakeFlags = [
     "-DBUILD_CONFIG=mysql_release"

